[2025-02-12T21:09:47.808+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T21:09:47.832+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_table_postgres_dag.create_tables_raw_orders manual__2025-02-12T21:09:44.909612+00:00 [queued]>
[2025-02-12T21:09:47.842+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_table_postgres_dag.create_tables_raw_orders manual__2025-02-12T21:09:44.909612+00:00 [queued]>
[2025-02-12T21:09:47.844+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-12T21:09:47.859+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_tables_raw_orders> on 2025-02-12 21:09:44.909612+00:00
[2025-02-12T21:09:47.868+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2018) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T21:09:47.870+0000] {standard_task_runner.py:72} INFO - Started process 2023 to run task
[2025-02-12T21:09:47.868+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'create_table_***_dag', 'create_tables_raw_orders', 'manual__2025-02-12T21:09:44.909612+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/create_table_***_dag.py', '--cfg-path', '/tmp/tmpwa9fnxl1']
[2025-02-12T21:09:47.871+0000] {standard_task_runner.py:105} INFO - Job 19: Subtask create_tables_raw_orders
[2025-02-12T21:09:47.890+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-12T21:09:47.937+0000] {task_command.py:467} INFO - Running <TaskInstance: create_table_postgres_dag.create_tables_raw_orders manual__2025-02-12T21:09:44.909612+00:00 [running]> on host 9718839f2a7d
[2025-02-12T21:09:48.242+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/utils/log/secrets_masker.py:84 DeprecationWarning: The sensitive_variable_fields option in [admin] has been moved to the sensitive_var_conn_names option in [core] - the old setting has been used, but please update your config.
[2025-02-12T21:09:48.278+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='create_table_***_dag' AIRFLOW_CTX_TASK_ID='create_tables_raw_orders' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T21:09:44.909612+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-12T21:09:44.909612+00:00'
[2025-02-12T21:09:48.279+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T21:09:48.329+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T21:09:48.339+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T21:09:48.340+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=create_table_***_dag, task_id=create_tables_raw_orders, run_id=manual__2025-02-12T21:09:44.909612+00:00, execution_date=20250212T210944, start_date=20250212T210947, end_date=20250212T210948
[2025-02-12T21:09:48.407+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T21:09:48.424+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T21:09:48.426+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
